pkg/environment/environment.go

```diff
@@ -65,33 +69,18 @@ func MustGetKrewPathsFromEnvs(envs []string) KrewPaths {
 		Index:    filepath.Join(base, "index"),
 		Install:  filepath.Join(base, "store"),
 		Download: filepath.Join(os.TempDir(), "krew"),
+		Bin:      filepath.Join(base, "bin"),
 	}
 }
 
-func getKubectlPluginsPath(envs []string) string {
-	envvars := parseEnvs(envs)
-
-	// Look for "${KUBECTL_PLUGINS_PATH}"
-	if path, ok := envvars["KUBECTL_PLUGINS_PATH"]; ok && path != "" {
-		return path
-	}
-
-	// TODO(lbb): make os call testable! (dep injection)
-	// Look for "${XDG_DATA_DIRS}/kubectl/plugins"
-	if path := envvars["XDG_DATA_DIRS"]; path != "" {
-		fullPath := filepath.Join(path, "kubectl", "plugins")
-		stat, err := os.Stat(fullPath)
-		if err == nil && stat.IsDir() {
-			return fullPath
-		}
-	}
-
+func getKubectlPluginsPath(_ []string) string {
 	// Golang does no '~' expansion
 	return filepath.Join(homedir.HomeDir(), ".kube", "plugins")
```

Comment:
(copying from the disappeared thread)
I think now that plugins don't have to be in ~/.kube/plugins, do you think we should start using ~/.krew ?

People `rm -rf ~/.kube` to reset their kubectl settings all the time. (I did that myself accidentally the other day).

Reply:
I'd like to keep it for now (dev-phase).
We should decide where to install krew, as it is also required to install plugin binaries into the PATH. We should maybe introduce KREW_PATH and KREW_BIN variable.

Reply:
maybe just convert this to a method that doesn't get any args for now.

Reply:
also maybe if this method is untested, we can just delete it.